build/includes/terraform.mk

```diff
@@ -27,14 +27,14 @@ terraform-clean:
 
 # Creates a cluster and install release version of Agones controller
 # Version could be specified by AGONES_VERSION
-# All Feature gates are disabled
+# Alpha Feature gates are disabled
 gcloud-terraform-cluster: GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT ?= 4
 gcloud-terraform-cluster: GCP_CLUSTER_NODEPOOL_MACHINETYPE ?= n1-standard-4
 gcloud-terraform-cluster: AGONES_VERSION ?= ''
 gcloud-terraform-cluster: GCP_TF_CLUSTER_NAME ?= agones-tf-cluster
 gcloud-terraform-cluster: LOG_LEVEL ?= debug
 gcloud-terraform-cluster: $(ensure-build-image)
-gcloud-terraform-cluster: FEATURE_GATES := ""
+gcloud-terraform-cluster: FEATURE_GATES := "ContainerPortAllocation=true"
```

Comment:
This shouldn't be required, if ContainerPortAllocation on by default?

(If it is, then that's a bug I think?)

Reply:
You are right, it seems it's redundant.